From ee8e820df9bb345565eeccf249be4bbfc4893307 Mon Sep 17 00:00:00 2001
From: Marek Szyprowski <m.szyprowski@samsung.com>
Date: Tue, 4 Mar 2014 12:30:01 +0100
Subject: [PATCH 1287/1302] ARM: odroidx2: update defconfig

Added drivers for MALI GPU, DMAbuf, DMAbuf-sync, exynos drm iommu and
cpufreq.

Signed-off-by: Marek Szyprowski <m.szyprowski@samsung.com>
Change-Id: I427332db24aa52daf4d4addbf9fbad8bc116b62e
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/configs/tizen_odroidx2_defconfig | 45 ++++++++++++++++++++++++++-----
 1 file changed, 39 insertions(+), 6 deletions(-)

diff --git a/arch/arm/configs/tizen_odroidx2_defconfig b/arch/arm/configs/tizen_odroidx2_defconfig
index 79fdc8c..5de4ff2 100644
--- a/arch/arm/configs/tizen_odroidx2_defconfig
+++ b/arch/arm/configs/tizen_odroidx2_defconfig
@@ -523,8 +523,35 @@ CONFIG_AUTO_ZRELADDR=y
 #
 # CPU Frequency scaling
 #
-# CONFIG_CPU_FREQ is not set
-# CONFIG_CPU_IDLE is not set
+CONFIG_CPU_FREQ=y
+CONFIG_CPU_FREQ_TABLE=y
+CONFIG_CPU_FREQ_GOV_COMMON=y
+CONFIG_CPU_FREQ_STAT=y
+# CONFIG_CPU_FREQ_STAT_DETAILS is not set
+CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
+# CONFIG_CPU_FREQ_DEFAULT_GOV_USERSPACE is not set
+# CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND is not set
+# CONFIG_CPU_FREQ_DEFAULT_GOV_CONSERVATIVE is not set
+# CONFIG_CPU_FREQ_DEFAULT_GOV_LAB is not set
+CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
+# CONFIG_CPU_FREQ_GOV_POWERSAVE is not set
+# CONFIG_CPU_FREQ_GOV_USERSPACE is not set
+CONFIG_CPU_FREQ_GOV_ONDEMAND=y
+# CONFIG_CPU_FREQ_GOV_CONSERVATIVE is not set
+# CONFIG_CPU_FREQ_GOV_LAB is not set
+
+#
+# ARM CPU frequency scaling drivers
+#
+CONFIG_ARM_EXYNOS_CPUFREQ=y
+CONFIG_ARM_EXYNOS4210_CPUFREQ=y
+CONFIG_ARM_EXYNOS4X12_CPUFREQ=y
+# CONFIG_ARM_EXYNOS5250_CPUFREQ is not set
+# CONFIG_ARM_KIRKWOOD_CPUFREQ is not set
+CONFIG_CPU_IDLE=y
+# CONFIG_CPU_IDLE_MULTIPLE_DRIVERS is not set
+CONFIG_CPU_IDLE_GOV_LADDER=y
+CONFIG_CPU_IDLE_GOV_MENU=y
 # CONFIG_ARCH_NEEDS_CPU_IDLE_COUPLED is not set
 
 #
@@ -718,7 +745,7 @@ CONFIG_REGMAP_I2C=y
 CONFIG_REGMAP_MMIO=y
 CONFIG_REGMAP_IRQ=y
 CONFIG_DMA_SHARED_BUFFER=y
-# CONFIG_DMABUF_SYNC is not set
+CONFIG_DMABUF_SYNC=y
 CONFIG_CMA=y
 CONFIG_CMA_DEBUG=y
 
@@ -1868,7 +1895,13 @@ CONFIG_DVB_AU8522_V4L=y
 #
 # ARM GPU Configuration
 #
-# CONFIG_MALI400 is not set
+CONFIG_MALI400=y
+# CONFIG_MALI400_DEBUG is not set
+# CONFIG_MALI400_PROFILING is not set
+CONFIG_MALI400_UMP=y
+# CONFIG_UMP_DEBUG is not set
+CONFIG_MALI_DVFS=y
+CONFIG_SLP_MALI_DBG=y
 CONFIG_DRM=y
 CONFIG_DRM_KMS_HELPER=y
 # CONFIG_DRM_LOAD_EDID_FIRMWARE is not set
@@ -1880,8 +1913,8 @@ CONFIG_DRM_KMS_HELPER=y
 # CONFIG_DRM_I2C_SIL164 is not set
 # CONFIG_DRM_I2C_NXP_TDA998X is not set
 CONFIG_DRM_EXYNOS=y
-# CONFIG_DRM_EXYNOS_IOMMU is not set
-# CONFIG_DRM_EXYNOS_DMABUF is not set
+CONFIG_DRM_EXYNOS_IOMMU=y
+CONFIG_DRM_EXYNOS_DMABUF=y
 CONFIG_DRM_EXYNOS_FIMD=y
 CONFIG_DRM_EXYNOS_HDMI=y
 # CONFIG_DRM_EXYNOS_VIDI is not set
-- 
1.8.3.2

